interfaces/builtin/basedeclaration.go

```diff
@@ -68,6 +68,11 @@ slots:
       slot-snap-type:
         - core
     deny-auto-connection: true
+  dcdbas-control:
+    allow-installation:
+      slot-snap-tupe:
+        - gadget
+    deny-auto-connection: true
```

Comment:
By specifying the slot type as 'gadget' here, you are saying that only gadgets will supply the slot (eg, perhaps through auto-detection or other means) but there is no slot side support in this interface.

I suspect you intended to have this be an implicit slot provided by the core snap. If so, please also update snap/implicit.go and change the 'slot-snap-type' to '- core'. If not, please describe how you intend to provide a slot implementation of this interface via a gadget snap.